src/components/pages/home/elements/WeatherBlock.js

```diff
@@ -1,12 +1,44 @@
 import React, { Component } from 'react';
+import WeatherTitle from './weatherTitle/WeatherTitle';
+import WeathewrAttributes from './weatherAttributes/WeatherAttributes';
+import getMarsWeatherData from '../../../../api/inSightApiClient';
 import '../HomePage.css';
 import PageNavButton from '../../../shared/PageNavButton';
 
 export default class WeatherBlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { weather: this.dummyValues() };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    getMarsWeatherData().then(weather => (this.setState({
+      weather: weather.slice(weather.length - 1)[0],
+    })
+    ));
+  }
+
+  dummyValues() {
+    return ({
+      earthDate: null,
+      sol: null,
+      pressure: {avg: null, max: null, min: null},
+      season: null,
+      temp: {avg: null, max: null, min: null},
+      wind: {avg: null, max: null, min: null, dir: null},
+    });
+  }
```

Comment:
Instead of having dummy values checked into the code we can construct the component with state `{ weather: null }` and only render the components that require the weather state once the weather has been loaded.

Have a chat to Anya and George about using the ternary operator to conditionally render components!